### DISCUSSION TITLE
crontab start of poetry

### DISCUSSION THREAD

**Discussion by mrosseel** [WimDeMeester/eFinder#16] [General]
Saw this comment somewhere (but now it's gone so either I missed it or you found a solution ) that it's a bit harder to start the app with crontab+poetry, I think the accepted solution is creating a shell script for startup, see discussion here:

https://github.com/python-poetry/poetry/issues/2179

**Comment by WimDeMeester** [comment]
This is what I have put in my crontab:
`PATH=/home/efinder/.local/bin:/home/efinder/.local/bin:/usr/local/astrometry/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/games:/usr/games
DISPLAY=:0
@reboot sleep 20 && (cd /home/efinder/Solver ; /home/efinder/.local/bin/poetry shell ; python /home/efinder/Solver/eFinder.py > /home/efinder/logs.txt 2>&1)
`

I guess it is a good plan to put this in a shell script for start up.  I'll do so later this week.

**Reply by WimDeMeester** [reply]
Updated this in the documentation.
